### ISSUE TITLE
[UI] Display frozen state of utxos more prominently

### ISSUE THREAD

**Issue by RequestPrivacy** [sparrowwallet/sparrow#348]
Currently, if you freeze an utxo this state is directly obvious by looking at the `UTXOs` tab by the means of changing the text colour of said utxo to gray. In all other tabs it isn't so easy to identify a frozen utxo though.

To enable an easy oversight of the state of an utxo it could be valuable to add a little icon (e.g. a snowflake) at all tabs where utxos are displayed. At the beginning of the "value" field could be a possibly good place.

A screenshot from the docs for clarification.

![sparrow_frozen](https://user-images.githubusercontent.com/72392544/145205780-6164803f-5d44-47c5-963c-1dd5aeb33ec6.png)

As a side note I also wonder if a frozen utxo should still be counted to the total balance (both BTC and fiat amount wise)? What are the typical use cases and the implications for them? What is the sentiment regarding this point? 

It could become confusing for the user though if he thinks the balance is zero and he deletes the wallet (and possibly seed) but there are still frozen utxos.

**Comment by craigraw** [comment]
> To enable an easy oversight of the state of an utxo it could be valuable to add a little icon (e.g. a snowflake) at all tabs where utxos are displayed. At the beginning of the "value" field could be a possibly good place.

The only place UTXOs are displayed is in the UTXOs tab - in the screenshot above, transactions are listed (and their inputs and outputs relevant to the wallet when expanded). So it wouldn't be correct to indicate that these are frozen.

> As a side note I also wonder if a frozen utxo should still be counted to the total balance (both BTC and fiat amount wise)? What are the typical use cases and the implications for them? What is the sentiment regarding this point?

I believe freezing a UTXO should not deduct it from the balance. There are multiple use cases for freezing a UTXO, including avoiding spending dust attack outputs, but also preventing a UTXO from mixing (say to increase the mix count on other UTXOs). The point on deleting a zero balance wallet accidentally is valid too.

**Comment by RequestPrivacy** [comment]
You're right, I wasn't precise. Although one could argue that at least a receive transaction is from a user standpoint almost identical to an utxo. But what about displaying the frozen state in the UTXO tab? 

Regarding the balance I tend to agree - the frozen utxos shouldn't be deducted from the balance. Or at least a little caution sign which warns people that the balance just represents the unfrozen state of the utxo set.

**Comment by craigraw** [comment]
> You're right, I wasn't precise. Although one could argue that at least a receive transaction is from a user standpoint almost identical to an utxo. But what about displaying the frozen state in the UTXO tab?

In the specific case of all the outputs in the transaction being frozen, it is somewhat analogous, but it still feels uncomfortable to me to mark a transaction as "frozen". In the UTXOs tab, frozen UTXOs are indicated through the grey/disabled text in the frozen row of the table.

> Or at least a little caution sign needs to be present to warn people that the balance just represents the unfrozen state of the wallets utxo set.

Difficult to get this right - in the case of frozen dust UTXOs, one wouldn't want a caution symbol on the Transactions tab drawing the users attention, IMO - it's too much UI distraction for something relatively minor. They would have little effect on the balance in any case.

**Comment by MatheyBTC** [comment]
This remains relevant in 2026.

In my current version (2.5.3), frozen UTXOs are not clearly marked: the snowflake icon is missing (not visible), and the only indication is a slightly dimmed row—a detail that can easily go unnoticed.

In contrast, the address reuse warning (a red exclamation mark icon) is immediately visible and unmistakable. The decision to freeze funds is just as important for coin control and deserves the same level of visual clarity: a clear icon (such as a padlock, a yellow alert, or a snowflake).

I have attached a screenshot of my UTXO view where the frozen status is not visually distinguishable.

<img width="1041" height="350" alt="Image" src="https://github.com/user-attachments/assets/efba6101-258b-445a-96dc-072fe87546db" />

**Comment by craigraw** [comment]
I would argue that the frozen row above is not "slightly" dimmed - it seems quite clearly different from the others. But I will consider this.
